### PR TITLE
How it works

### PR DESCRIPTION
**Pull request by hicondron**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -15,8 +15,9 @@ body {
 
 img.how_picture{
     float: right;
```
Comment (hicondron) [inline css/style.css]: add padding to picture

File: css/style.css [hunk 2]
```diff
@@ -32,5 +33,9 @@ img.how_picture{
 }
 
 .how_lists h1 {
```
Comment (hicondron) [inline css/style.css]: padding padding to lis

File: index.html [hunk 3]
```diff
@@ -15,43 +15,46 @@
         <header id="header"></header>
 
         <!-- HOW IT WORKS SECTION -->
```
Comment (hicondron) [inline index.html:52]: move </section> to end of section

**Review by hicondron — COMMENTED** [review]
add styling to picture
move </section> to end of section in HTML

**Review by BachAmadou — APPROVED** [review]

**Pull request merged**